interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/bind-to-data/data-binding-modes.md

```diff
@@ -1 +1,118 @@
-[comment]: <> (This topic is intended for the Conversion Dialog. Explain when a user should select to convert its report from data bindings to expressions)
\ No newline at end of file
+---
+title: Data Binding Modes
+author: Anna Gubareva
+---
+# Data Binding Modes
+
+## Conversion Dialog
+
+The following dialog can appear when **expression bindings** are enabled in the Report Designer, and you [open an existing report](../open-reports.md) that uses standard **data bindings**:
```

Comment:
This dialog can appear (possibility), but if it does not you should check the mode? Does this mean it should appear?

Reply:
This dialog appears only in one case: when expression bindings are enabled in the Report Designer and a user opens a report with data bindings. 

Since it's unclear that the dialog is shown only in specific cases, we decided to change the section order to start with binding mode overview.
- Expression Bindings
- Data Bindings
- Conversion Dialog
- Binding Mode Comparison

__________
Conversion Dialog

If expression bindings(link to the first section) are enabled in the Report Designer, and you open an existing report that uses standard data bindings(link to the second section), the following dialog can appear:

image

See the section below to learn how to migrate to use expressions instead of data bindings.

Reply:
Perhaps you should say 'This dialog appears only when expression bindings are enabled in the Report Designer and a user opens a report with data bindings'?

OR

The following dialog appears if expression bindings (link to the first section) are enabled in the Report Designer, and you open an existing report that uses standard data bindings (link to the second section):

image

See the section below for information on how to use expressions instead of data bindings.